docs/designers-developers/developers/filters/block-filters.md

```diff
@@ -67,6 +67,40 @@ A filter that applies to the result of a block's `save` function. This filter is
 
 The filter's callback receives an element, a block type and the block attributes as arguments. It should return an element.
 
+This example below wraps the existing save element inside of a wrapper element.
+
+_Example:_
+
+Adding a wrapper to core quote blocks
+
+{% codetabs %}
+{% ES5 %}
+```js
+var createElement = wp.element.createElement;
+const quoteSaveFilter = (el, type, attributes) => {
```

Comment:
These code examples should follow the coding standards (spaces between parentheses, etc).

Also, this is not ES5 code despite being part of the `{% ES5 %}` code block (`const` declarations).